src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (robot[key] in invertedRobot) {
+      return null;
+    }
+
+    invertedRobot[robot[key]] = key;
```

Comment:
When assigning the key to the value in `invertedRobot`, it's a good practice to ensure that the value is a string, as object keys are always strings in JavaScript. If the original key in the `robot` object is not a string (e.g., a number), it will be coerced to a string when used as a key in `invertedRobot`. If this behavior is intended, then the code is correct. Otherwise, consider converting the key to a string explicitly if needed.